packages/p2p/src/stores/sendbird-store.js

```diff
@@ -237,7 +237,7 @@ export default class SendbirdStore extends BaseStore {
     markMessagesAsRead(should_check_scroll) {
         if (!this.active_chat_channel) return;
 
-        if (document.hasFocus()) {
+        if (document.hasFocus() && ((isMobile() && this.should_show_chat_modal) || !isMobile())) {
```

Comment:
**nit**: Instead of using ```!isMobile``` we should use ```isDesktop```
```suggestion
        if (document.hasFocus() && ((isMobile() && this.should_show_chat_modal) || isDesktop())) {
```